src/api/handlers/guild.ts

```diff
@@ -769,6 +769,16 @@ export function getGuild(guildID: string, counts = true) {
 }
 
 /** Returns the guild template if it exists */
+export function getTemplate(templateCode: string) {
+  return RequestManager.get(endpoints.GUILD_TEMPLATE(templateCode)) as Promise<
+    GuildTemplate
+  >;
+}
+
+/** 
+ * Returns the guild template if it exists 
+ * @deprecated This function no longer works
```

Comment:
How is this deprecated?

Reply:
Better said this does not work. Discord will throw an error. Also endpoints are not the same.